bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BFunctionType.java

```diff
@@ -135,4 +138,14 @@ public String getAnnotationKey() {
     public boolean isReadOnly() {
         return true;
     }
+
+    public List<BString> getParamAnnotationKeys() {
```

Comment:
Wondering if we should have a public method in `AnnotatableType` that returns all the annotations, and move this logic to wherever this specific set of keys is needed.

@Kishanthan, thoughts please?

Reply:
At least a method to get all keys would be sufficient (`getAnnotationKeys`) if you think the function is quite specif.

Reply:
Yes, lets add the new method to the AnnotatableType class as it is the common class for all the annotatable types.

Reply:
Fixed via 3b23fbf